console/src/components/Services/ApiExplorer/OneGraphExplorer/OneGraphExplorer.js

```diff
@@ -191,7 +192,12 @@ class OneGraphExplorer extends React.Component {
 
     const { renderGraphiql } = this.props;
 
-    const explorer = (
+    const explorer = this.props.loading ? (
+      <div style={{ width: explorerWidth, marginTop: 170 }}>
+        <Spinner />
+        <div style={{ marginLeft: 90 }}>Loading...Schema</div>
+      </div>
+    ) : (
```

Comment:
Could you leave only Spinner, and remove `Loading ...` text?

Also, apply those styles to the parent div:
```
    width: 300px;
    display: flex;
    align-items: center;
    height: 100%;
```

There are already classes for it so you'd need to use:
* .wd300Px
* .display_flex
* .height100

```jsx
<div className={`${styled.height100} ${styles.display_flex} ${styles.wd300Px}`}> ....
```